apps/molgenis-components/src/components/layout/Molgenis.vue

```diff
@@ -0,0 +1,210 @@
+        >
+          {{ session.manifest.SpecificationVersion }} </a
+        >.
+        <span v-if="session.manifest.DatabaseVersion">
+          Database version: {{ session.manifest.DatabaseVersion }}.
+        </span>
+      </span>
+    </MolgenisFooter>
+  </div>
+</template>
+
+<script>
+import MolgenisMenu from "./MolgenisMenu.vue";
+import MolgenisSession from "./MolgenisSession.vue";
+import MolgenisFooter from "./MolgenisFooter.vue";
+import Breadcrumb from "./Breadcrumb.vue";
+
+/**
+ Provides wrapper for your apps, including a little bit of contextual state, most notably 'account' that can be reacted to using v-model.
+ */
+export default {
+  components: {
+    MolgenisSession,
+    MolgenisMenu,
+    MolgenisFooter,
+    Breadcrumb,
+  },
+  props: {
+    menuItems: {
+      type: Array,
+      default: [
+        { label: "Tables", href: "tables", role: "Viewer" },
+        {
+          label: "Schema",
+          href: "schema",
+          role: "Manager",
+        },
+        {
+          label: "Up/Download",
+          href: "updownload",
+          role: "Editor",
+        },
+        {
+          label: "Graphql",
+          href: "graphql-playground",
+          role: "Viewer",
+        },
+        {
+          label: "Settings",
+          href: "settings",
+          role: "Manager",
+        },
+        {
+          label: "Help",
+          href: "docs",
+          role: "Viewer",
+        },
+      ],
+    },
+    title: String,
+    showCrumbs: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  data: function () {
+    return {
+      session: null,
+      cssURL: null,
+      logoURL: null,
+      cssLoaded: false,
+      fullscreen: false,
+      timestamp: Date.now(),
+    };
+  },
+  computed: {
+    schemaUrlsForCrumbs() {
+      var result = {};
+      if (this.session && this.session.schemas) {
+        //all databases
+        result["list all databases"] = "/";
+        this.session.schemas.forEach((s) => {
+          result[s] = "../../" + s; // all paths are of form /:schema/:app
+        });
+      }
+      return result;
+    },
+    crumbs() {
+      this.$route;
+      let path = decodeURI(
+        window.location.pathname.replace(location.search, "")
+      ).split("/");
+      let url = "/";
+      let result = {};
+      if (window.location.pathname != "/apps/central/") {
+        path.forEach((el) => {
+          if (el != "") {
+            url += el + "/";
+            result[el] = url;
+          }
+        });
+      }
+      if (this.$route) {
+        path = decodeURI(location.hash.split("?")[0]).substr(1).split("/");
+        url += "#";
+        path.forEach((el) => {
+          if (el != "") {
+            url += "/" + el;
+            result[el] = url;
+          }
+        });
+      }
+      return result;
+    },
+    logo() {
+      if (this.logoURL) return this.logoURL;
+      else
+        return "/apps/molgenis-components/assets/img/molgenis_logo_white.png";
```

Comment:
function should probably be called logoURL or logoURLString , and path should not be part of it , just keep de main img files an img folder